### PR TITLE
Added ColMarTech for Correspondents

### PR DESCRIPTION
**Pull request by Tunguso4ka**
## About the PR

Added Combat Correspondent's equipment rack.

## Media

зображення

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

**Comment by QuietlyWhisper** [comment]
Do you want to add anything from #3250? I'll leave this open

**Comment by Kadeo64** [comment]
whenever this gets merged i hope the mapping updates that add this follow suit quickly.

Also, would it be possible to add a microphone and hand labeler?

**Comment by Tunguso4ka** [comment]
> Also, would it be possible to add a microphone and hand labeler?

correspondents round spawn with microphone.

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/combat_correspondent.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    - name: Refills
+      entries:
+      # /obj/item/device/camera 10
+      # /obj/item/device/camera_film 5
+      # /obj/item/device/toner 5
+      # /obj/item/storage/box/tapes 15
+      - id: CMPaper
+        amount: 30
+      - id: CMHandLabeler
+        amount: 3
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/combat_correspondent.yml:56]: This is supposed to be 10 cameras, 5 camera films, 5 toners, 15 regulation tapes (none of those we have I think), 10 paper bins

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/combat_correspondent.yml (hunk 1 above)
Comment (DrSmugleaf) [reply Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/combat_correspondent.yml:56]: Kadeo64 is going to need to settle for no CMHandLabelers, there's some in medical and Req so we don't have to AHelp him for trying to use it to bypass the fix for xenos being able to read papers

Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/combat_correspondent.yml:56]:
```suggestion
```

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]
Thanks for the PR!

**Pull request merged**